sql/pgmq--0.19.0--0.20.0.sql

```diff
@@ -0,0 +1,10 @@
+DO $$
+DECLARE
+    qname TEXT;
+BEGIN
+    FOR qname IN (SELECT queue_name FROM public.pgmq_meta)
+    LOOP
+        EXECUTE format('ALTER TABLE %I RENAME COLUMN deleted_at TO archived_at', 'pgmq_' || qname || '_archive');
+        EXECUTE format('ALTER INDEX %I RENAME TO %I', 'deleted_at_idx_' || qname, 'archived_at_idx_' || qname);
```

Comment:
This index name isn't prefixed, all other things are. Could be good to take the opportunity to do so.

Reply:
Maybe we can leave the prefix off? Once we move everything to the pgmq schema we could drop the prefix on everything..